schedule_lessons/dashboard/static/dashboard/js/relationships.js

```diff
@@ -1,6 +1,10 @@
 $(document).ready(function () {
   $('[data-toggle="tooltip"]').tooltip();
 
+  $("#secondNavBtn").click(function(){
+    $("#navbarSupportedContent").slideToggle("fast");
```

Comment:
question: what does passing in 'fast' do? I know it is the duration of the toggle right?

Reply:
Yeah, it toggles the menu faster. It matches the speed of the Bootstrap navbar.

Reply:
cool, good to know!